frontend/app/controllers/Info.scala

```diff
@@ -152,6 +152,14 @@ trait Info extends Controller {
     Ok(views.html.info.patron(pageInfo, pageImages))
   }
 
+  def offersAndCompetitions = CachedAction { implicit request =>
+    val results = GuardianContentService.offersAndCompetitionsContent.map(ContentItemOffer)
+      .filter(_.content.fields.map(_("membershipAccess")).isEmpty)
+      .filter(!_.content.webTitle.startsWith("EXPIRED"))
+      .filter(_.imgOpt.nonEmpty)
```

Comment:
Yeah, totally agree. Wasn't my intended scope (just moving the template), but no reason I can't leave the campsite a little better than I found it. Happy to clean this up as part of this PR.